DSCResources/MSFT_xSqlServerInstall/MSFT_xSqlServerInstall.schema.mof

```diff
@@ -19,5 +21,10 @@ class MSFT_xSqlServerInstall : OMI_BaseResource
     [write, Description("Specify the startup account for the SQL server service.")] string SvcAccount;
     [write, Description("Specify logins to be members of the sysadmin role.")] string SysAdminAccounts;
     [write, Description("Specify the account for SQL server agent service.")] string AgentSvcAccount;
+    [write, Description("Specify the default collation for SQL server")] string SqlCollation;
```

Comment:
A value map would be useful for SqlCollation, if you are not going to add it now, we should open an issue for it.